src/test/unit/algorithms/Serialization.cpp

```diff
@@ -34,11 +36,13 @@
 #include <nupic/os/Timer.hpp>
 #include <nupic/utils/Random.hpp>
 
+namespace testing {
+
 using namespace std;
 using namespace nupic;
 using namespace nupic::algorithms::spatial_pooler;
 
-void testSP() {
+TEST(Serialization, testSP) {
```

Comment:
I think youve removed/changed this file (former prototest), whichever comes first, we'll have to update the merge conflict.